src/language/branches.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Branches 
+description: Learn how to use branches to control the flow of your Dart code.
+---
+
+You can control the flow of your Dart code using branches:
+
+-   `if`, `case`, and `else`
+-   `switch` and `case` statements
+-   `switch` and `case` expressions
+
+Other methods of manipulating control flow in Dart are:
+
+-   [`for` loops][]
+-   [`while` and `do while` loops][]
+-   [`break` and `continue`][break]
+-   [`assert`][]
+
+You can also affect control flow using [Exceptions][], like `try-catch` and `throw`.
+
+## If else
```

Comment:
I'd either leave off `else` here or add it to `## If case` below since both of them can either have or omit the `else` clause. Or maybe do:

```
## If and If-else

...

## If-case and If-case-else
```